schema/schemaspec/schemahcl/hcl.go

```diff
@@ -306,6 +310,15 @@ func hclType(spec *schemaspec.TypeSpec, typ *schemaspec.Type) (string, error) {
 	return fmt.Sprintf("%s(%s)", typ.T, strings.Join(args, ",")), nil
 }
 
+func findAttr(attrs []*schemaspec.Attr, k string) (*schemaspec.Attr, bool) {
+	for _, atr := range attrs {
+		if atr.K == k {
+			return atr, true
+		}
+	}
+	return nil, false
+}
```

Comment:
```suggestion
func hasAttr(attrs []*schemaspec.Attr, k string) (*schemaspec.Attr, bool) {
	for _, attr := range attrs {
		if attr.K == k {
			return attr, true
		}
	}
	return nil, false
}
```

Let's stay consistent with other parts.

Reply:
I think it's odd for a function that returns the value to be called `hasX` which usually only returns a bool. Don't you think?